<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pagePlayer</class>
 <widget class="QWidget" name="pagePlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QLabel" name="labelPlayerPix">
         <property name="minimumSize">
          <size>
           <width>220</width>
           <height>275</height>
          </size>
         </property>
         <property name="text">
          <string>球员图片</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelPlayerName">
         <property name="text">
          <string>球员名字</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QPushButton" name="buttonAssisting">
         <property name="text">
          <string>助攻/抢断/盖帽</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="buttonFalut">
         <property name="text">
          <string>失误/犯规</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="buttonScore">
         <property name="text">
          <string>分数</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="buttonVictory">
         <property name="text">
          <string>胜负</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="buttonShoot">
         <property name="text">
          <string>投篮</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="buttonThree">
         <property name="text">
          <string>三分</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="buttonPenalty">
         <property name="text">
          <string>罚球</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="buttonBackboard">
         <property name="text">
          <string>篮板</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>580</width>
        <height>69</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <widget class="QWidget" name="pageShoot">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QGridLayout" name="layoutShoot"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageThree">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QGridLayout" name="layoutThree"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pagePenalty">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QGridLayout" name="layoutPenalty"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageBackboard">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QGridLayout" name="layoutBackboard"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageAssisting">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QGridLayout" name="layoutAssisting"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageFalut">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QGridLayout" name="layoutFalut"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageScore">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QGridLayout" name="layoutScore"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageVictory">
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QGridLayout" name="layoutVictory"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
